dkms

```diff
@@ -1414,10 +1414,29 @@ clean_build()
     rm -rf "$dkms_tree/$module/$module_version/build"
 }
 
+sign_build()
+{
+    local base_dir="$dkms_tree/$module/$module_version/$kernelver/$arch"
+    if type update-secureboot-policy >/dev/null 2>&1; then
+        echo $"Signing module:"
+        SHIM_NOTRIGGER=y update-secureboot-policy --new-key
+        for ko in `find "$base_dir/module/" -name "*.ko" -print`;
+        do
+            echo " - $ko"
+            kmodsign sha512 \
+                /var/lib/shim-signed/mok/MOK.priv \
```

Comment:
Similarly this makes an assumption about the directory structure being used.  I think that needs to be looked up dynamically or at least test that this structure already exists.

Reply:
The idea is to see if other distro starts using this as Ubuntu did, so in this case we can extend (or I'm sure they will do it)...